Dockerfile.scons_cache

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3
+
+COPY ./scons_cache /tmp/scons_cache
```

Comment:
I don't want to pollute the root with a file like this. let's either move or generate on the fly